README.md

```diff
@@ -9,7 +9,7 @@ login functionality in email and newsletter links.
 The library is published as a [package][] and is installable via [Composer][]:
 
 ```
-$ composer require jmikola/auto-login=~1.0
+$ composer require jmikola/auto-login
```

Comment:
It looks like Composer automatically adds a version qualifier (e.g. "^1.2") when installing, so we can definitely remove this. That will still ensure that only BC upgrades are permitted, which was the original intention. IIRC, Composer used to fall back to "*" if a version wasn't specified.

Reply:
Hm, not sure about falling back to `*`, but looks like not relevant to this repo already since this package has some releases on Packagist